moveit_setup_assistant/moveit_setup_srdf_plugins/moveit_setup_framework_plugins.xml

```diff
@@ -0,0 +1,23 @@
+<library path="libmoveit_setup_srdf_plugins">
```

Comment:
No need to have the `lib` prefix, also this fixes a warning from `pluginlib.ClassLoader`
```suggestion
<library path="moveit_setup_srdf_plugins">
```